src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -413,20 +458,23 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
                 bool updatedSolution = false;
                 for (size_t i = 0; i < goalMotions_.size(); ++i)
                 {
-                    if (opt_->isCostBetterThan(goalMotions_[i]->cost, bestCost))
+                    if (opt_->isCostBetterThan(goalMotions_[i]->cost, bestCost_))
                     {
-                        bestCost = goalMotions_[i]->cost;
-                        bestCost_ = bestCost;
```

Comment:
Here both variables are also set to the same thing. These are the only two places that either variable are assigned, so therefore, we can just remove `bestCost`.